Tests/SwiftStdlibTests/StringExtensionsTests.swift

```diff
@@ -541,6 +541,15 @@ final class StringExtensionsTests: XCTestCase {
         XCTAssertTrue("email@mail.com".matches(pattern: "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,}"))
     }
 
+    func testMatchings() {
```

Comment:
Can you please add tests that will have multiple results? That way you're not just checking if the result `isEmpty` or `throws`.